### PR TITLE
Fixes EFF logo showing in Firefox

### PR DESCRIPTION
**Pull request by caseyyee**
image

Seems like Firefox didn't like the svg. Uses PNG instead.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h1>Get ready to Spot the Surveillance...</h1>
         <img id="congrats-card" src="assets/images/congrats-card-replay.png">
         <img id="drone-card"  src="assets/images/drone-card.png">
         <img id="intro-card" src="assets/images/intro-card.png">
-        <img id="eff-logo" src="assets/images/eff-logo-black.svg">
```
Comment (cvan) [inline index.html]: @lschatzkin: can we remove the `eff-logo-black.svg` from this repo?

File: index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h1>Get ready to Spot the Surveillance...</h1>
...
+        <img id="eff-logo" src="assets/images/eff-logo-black.png">
```
Comment (cvan) [inline index.html]: can you compress the PNG with `pngquant`? not a biggie, but was able to shave off 2 KB.

File: index.html (hunk 1 above)
Comment (lschatzkin) [reply index.html]: > @lschatzkin: can we remove the `eff-logo-black.svg` from this repo?

yes!

**Comment by caseyyee** [comment]
fantastic, thanks @lschatzkin !

**Pull request merged**